functions/src/api/auth/assertValidToken.spec.ts

```diff
@@ -0,0 +1,46 @@
+// tslint:disable-next-line
+import '../../testUtils/mockFirebaseAdminAndFunctions'
+import { firestoreGet } from '../../testUtils/firestoreStub'
+import { JWTData } from '../../types/JWT'
+import { assertValidToken } from './assertValidToken'
+import { ForbiddenError } from '../errors/HttpError'
+
+describe('assertValidToken', () => {
+    const token = 'tokenValue'
+    const jwtData: JWTData = {
+        uid: '69',
+        iat: 1544455,
+    }
+
+    it('should not throw exception because token is valid in DB', async () => {
```

Comment:
I would suggest to use affirmative sentences to describe tests. What about something like

```
it should confirm token validity in DB
```

Reply:
this is true for isTokenValid, but not for the assertValidToken, the meaning is different and it makes sens based on the method behavior.